pages/api/module/del.ts

```diff
@@ -9,11 +9,14 @@ const del = async (
     const { id } = req.query;
     const module = await Module.findByIdAndDelete(id);
 
-    if (!module)
-        return res
-            .status(404)
-            .send({ error: "The module with the given ID was not found." });
-    res.status(204).send({});
+    if (module) {
+        res.status(204).end();
+    } else {
+        res.status(404).send({
+            error: "The module with the given ID was not found.",
+        });
+    }
+    return;
```

Comment:
This resolved the warning `A body was attempted to be set with a 204 statusCode for /api/module/del?id=62001c8e6ed8641a8635697d, this is invalid and the body was ignored.`. Please let me know if the code looks ok here.